model/atmosphere/diffusion/src/icon4py/model/atmosphere/diffusion/stencils/apply_diffusion_to_w_and_compute_horizontal_gradients_for_turbulence.py

```diff
@@ -49,8 +50,8 @@ def _apply_diffusion_to_w_and_compute_horizontal_gradients_for_turbulence(
 ]:
     k = broadcast(k, (CellDim, KDim))
```

Comment:
we should remove all `k` fields before merge and possibly the horizontal ones as well (if we are using them in concat_where)